crud.py

```diff
@@ -8,28 +8,41 @@
 # manager
 def create_manager(db: Session, manager: schemas.ManagerCreate):
     db_manager = models.Manager(
-        manager_name = manager.manager_name,
```

Comment:
We should keep a username for the manager

Reply:
ok

Reply:
We want to pass the firstname, lastname (as you did below) and also the username, let's bring this line back as username